Apps/Playground/ios/Podfile

```diff
@@ -4,24 +4,20 @@ require_relative '../node_modules/@react-native-community/cli-platform-ios/nativ
 platform :ios, '12.0'
 
 target 'Playground' do
-  permissions_path = '../node_modules/react-native-permissions/ios'
-  pod 'Permission-Camera', :path => "#{permissions_path}/Camera.podspec"
-
   config = use_native_modules!
-
-  use_react_native!(:path => config["reactNativePath"])
-
-  target 'PlaygroundTests' do
+  use_react_native!(
+    :path => config[:reactNativePath],
+    # to enable hermes on iOS, change `false` to `true` and then install pods
+    :hermes_enabled => false
+  )
+  target 'RnDiffAppTests' do
```

Comment:
"RnDiffApp" is the name of the test app they use for the upgrade helper. Anywhere you see this needs to be replaced with "Playground".